.github/workflows/pr-title-linter.yml

```diff
@@ -16,15 +16,17 @@ jobs:
     steps:
       - name: Checkout Repo
         uses: actions/checkout@v4
-
-      - name: Setup
-        id: config
-        uses: ./.github/actions/init
         with:
-          turbo-signature: ${{ secrets.TURBO_REMOTE_CACHE_SIGNATURE_KEY }}
-          turbo-team: ${{ vars.TURBO_TEAM }}
-          turbo-token: ${{ secrets.TURBO_TOKEN }}
-          playwright-enabled: true # Must be present to enable caching on branched workflows
+          fetch-depth: 1
+          show-progress: false
+          sparse-checkout: |
+            commitlint.config.ts
+            packages/*/package.json
+            tsconfig.json
+          sparse-checkout-cone-mode: false
 
       - name: Lint Pull Request Title
-        run: echo "${{ github.event.pull_request.title }}" | npx commitlint
+        run: |
+          npm init --scope=clerk --yes
+          npm i --save-dev @commitlint/cli globby typescript --audit=false --fund=false
```

Comment:
Can we add a comment here and/or in the commitlint config noting the dependency between the packages installed here and those used in our commitlint config? If the commitlint is updated in some way we need to make sure the packages it relies on are installed in this workflow.